packages/playground/src/components/select_node.vue

```diff
@@ -276,27 +264,30 @@ async function loadNodes(farmId: number) {
     }
   }
 }
-async function validateNodeStoragePool(grid: GridClient, nodeId: number, disks: number[], rootFileSystemSize: number) {
+async function validateNodeStoragePool(validatingNode: INode | undefined) {
+  if (!validatingNode) return { message: "Node id is required." };
   farmManager?.setLoading(true);
   validator.value?.setStatus(ValidatorStatus.Pending);
   pingingNode.value = true;
   try {
-    await grid.capacity.checkNodeCapacityPool({
-      nodeId,
-      ssdDisks: disks,
-      rootfsDisks: [rootFileSystemSize],
+    const grid = await getGrid(profileManager.profile!);
+    await grid!.capacity.checkNodeCapacityPool({
```

Comment:
do an if(grid) instead of the forbidden null assertion. Also please remove unused imports on lines 82 and 83.

Reply:
done, we can enhance the error message later on https://github.com/threefoldtech/tfgrid-sdk-ts/issues/1341